prefab-cloud-ruby.gemspec

```diff
@@ -38,31 +40,43 @@ Gem::Specification.new do |s|
     "lib/prefab/config_helper.rb",
```

Comment:
We can remove `config_helper.rb`, I think.

Reply:
Yes! f979b50